test_scripts/Polices/build_options/031_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Computation_HTTP.lua

```diff
@@ -48,12 +49,34 @@ local f_name = os.tmpname()
 local ptu
 local sequence = { }
 local accuracy = 5
-local r_expected = { 11, 22, 33 }
+local r_expected = { 30, 31, 62, 93, 124} -- 155 is for finish retry sequence
 local r_actual = { }
-local attempts = 15
+local attempts = ( 30 + 31 + 62 + 93 + 124 )
 local timestamps = { }
 
 --[[ Local Functions ]]
+local function update_ptu()
+  if(ptu == nil) then
+    local config_path = commonPreconditions:GetPathToSDL()
+    local pathToFile = config_path .. 'sdl_preloaded_pt.json'
+
+    local file = io.open(pathToFile, "r")
+    local json_data = file:read("*all")
+    file:close()
+
+    ptu = json.decode(json_data)
+  end
+
+  ptu.policy_table.device_data = nil
```

Comment:
@istoimenova, Please add removing of _preloaded_pt_ and _preloaded_date_ parameters. Since they may be present in _sdl_preloaded_pt.json_ file